server/settings/publishers.py

```diff
@@ -787,11 +757,6 @@ class PublishersModel(BaseSettingsModel):
         title="Yeti Rig Settings"
     )
     # Model - START
-    ValidateModelName: ValidateModelNameModel = SettingsField(
-        default_factory=ValidateModelNameModel,
-        title="Validate Model Name",
-        section="Model",
-    )
     ValidateModelContent: ValidateModelContentModel = SettingsField(
         default_factory=ValidateModelContentModel,
         title="Validate Model Content",
```

Comment:
Sections must now be on the next entry
```suggestion
        title="Validate Model Content",
        section="Model"
```